lib/timeline/Core.js

```diff
@@ -393,13 +393,7 @@ Core.prototype._create = function (container) {
  */
 Core.prototype.setOptions = function (options) {
   if (options) {
-    // copy the known options
-    var fields = [
-      'width', 'height', 'minHeight', 'maxHeight', 'autoResize',
-      'start', 'end', 'clickToUse', 'dataAttributes', 'hiddenDates',
-      'locale', 'locales', 'moment', 'rtl', 'zoomKey', 'horizontalScroll', 'verticalScroll'
-    ];
-    util.selectiveExtend(fields, this.options, options); 
+    this.options = { ...this.options, ...options }
```

Comment:
I'm not sure if this is the richt way to go. This way we allow all options to be passed trough. This may be a security issue?

Reply:
Not really no. What is the security issue really? I can reach any option anywhere through <Component>.options

Reply:
I don't feel good about this. Why was this done like this in the first place?
Why not just fix the bug in the `selectiveExtend` function?